sr_robot_launch/README.md

```diff
@@ -38,51 +93,40 @@ Trying to read mapping for: /hand/mapping/1178
 
 In this case 1178 is the serial number of the hand.
 
-### Real Robot hand only
-
-To start the hand without an arm:
-
-```bash
-roslaunch sr_robot_launch right_srhand_ur10arm.launch sim:=false arm_ctrl:=false arm_trajectory:=false hand_serial:=1178
-```
-
-or, for the left hand
-
-```bash
-roslaunch sr_robot_launch left_srhand_ur10arm.launch sim:=false arm_ctrl:=false arm_trajectory:=false hand_serial:=1178
-```
-
-### Real Robots, using the normal (not limited) joint range
+#### Real Robots, using the normal (not limited) joint range
 
 By default the URDF used for the UR10 arm uses a limited range for the joints, as that helps moveit find a planning solution. But as that restricts the robot movements, the user might want to start the robots with the full joint range. To do that:
 
 ```bash
-roslaunch sr_robot_launch right_srhand_ur10arm.launch sim:=false robot_description:=`rospack find sr_multi_description`/urdf/right_srhand_ur10.urdf.xacro hand_serial:=1178
+roslaunch sr_robot_launch sr_right_ur10arm_hand.launch sim:=false robot_description:=`rospack find sr_multi_description`/urdf/right_srhand_ur10.urdf.xacro hand_serial:=1178
 ```
 
 or, for the left hand and arm
 
 ```bash
-roslaunch sr_robot_launch left_srhand_ur10arm.launch sim:=false robot_description:=`rospack find sr_multi_description`/urdf/left_srhand_ur10.urdf.xacro hand_serial:=1178
+roslaunch sr_robot_launch sr_left_ur10arm_hand.launch sim:=false robot_description:=`rospack find sr_multi_description`/urdf/left_srhand_ur10.urdf.xacro hand_serial:=1178
 ```
 
-### Hand with Biotacs
+#### Hand with Biotacs
```

Comment:
Maybe we can generalize this to "Hand tactile sensors"?
And have subsections for the different types.